src/Components/ProvisioningWizard/steps/SourceMissing/SourceMissing.test.js

```diff
@@ -70,4 +70,28 @@ describe('Source missing', () => {
       expect(directLink).toHaveAttribute('href', url);
     });
   });
+
+  describe('GCP', () => {
+    const image = {
+      ...gcpImage,
+      uploadStatus: { options: { image_name: 'cool-image' } },
+      uploadOptions: { accountIDs: 'example@redhat.com' },
+    };
+
+    test('has create source button', () => {
+      render(<SourceMissing image={image} />);
+
+      const sourcesLink = screen.getByRole('link', { name: 'Create Source' });
+      expect(sourcesLink).toHaveAttribute('href', '/preview/settings/sources/new');
+    });
+
+    test('renders direct link', () => {
+      render(<SourceMissing image={image} />);
+
+      const url = 'https://console.cloud.google.com/welcome';
```

Comment:
Do we have a ticket for this? I've noticed this for second time, but I guess we do not want to fix this now.
What I mean is opening a welcome page is not very meaningful prolly :) 
We shall have a ticket to remember tho :)